### PR TITLE
Light Mode Interaction States - List - RSC-1473

### PR DESCRIPTION
**Pull request by avnvbhatta**
https://issues.sonatype.org/browse/RSC-1473

### CODE REVIEW

File: gallery/visualtests/nxList.js [hunk 1]
```diff
@@ -27,11 +34,27 @@ describe('nx-list', function() {
...
 
   describe('Clickable links nx-list', function() {
     it('looks right with a row hovered',
         hoverTest(clickableLinksSelector, `${clickableLinksSelector} li:first-child a`));
+
+    it('looks right with a row focused',
+        focusTest(clickableLinksSelector, `${clickableLinksSelector} li:first-child a`));
+
+    it('looks right with a row clicked',
+        focusTest(clickableLinksSelector, `${clickableLinksSelector} li:first-child a`));
```
Comment (entroform) [inline gallery/visualtests/nxList.js:54]: Just wondering if this was supposed to be `clickTest`

**Comment by EmilyBermudez88** [comment]
The hover state of `.nx-list__item--clickable` needs to have its text change to `nx-color-text-dark` on hover :)

**Comment by avnvbhatta** [comment]
> The hover & 'selected' state of `.nx-list__item--clickable` needs to have its text change to `nx-color-text-dark` :)

Oo good catch. I see that it also needs to be the darker text on active as well. Changed here:
[4f054ab](https://github.com/sonatype/sonatype-react-shared-components/pull/1000/commits/4f054ab14ca4cfcb5400df5a5cb4d3609b10aa66)

**Comment by EmilyBermudez88** [comment]
> > The hover & 'selected' state of `.nx-list__item--clickable` needs to have its text change to `nx-color-text-dark` :)
> 
> Oo good catch. I see that it also needs to be the darker text on active as well. Changed here: [4f054ab](https://github.com/sonatype/sonatype-react-shared-components/commit/4f054ab14ca4cfcb5400df5a5cb4d3609b10aa66)

Good catch on the active text, I never noticed that. I just think you need to update the NxDescriptionList snapshots as well.

**Review by EmilyBermudez88 — APPROVED** [review]
LGTM! Just update the snapshots in https://github.com/sonatype/sonatype-react-shared-components/pull/1000#issuecomment-1463903215

**Pull request merged**